### PR TITLE
Documentation: statistical data

### PR DESCRIPTION
**Pull request by ljhwang**
Adds footnotes, note box, typo clean-up, and a few miscellaneous changes.
Still needs updates for section reference links and figure.

### CODE REVIEW

File: doc/sphinx/user/run-aspect/visualizing-results/stat-data.md [hunk 1]
```diff
@@ -6,11 +6,10 @@ In addition to the graphical output discussed above,
 ASPECT produces a statistics file that collects
 information produced during each time step. For the remainder of this section,
 let us assume that we have run ASPECT with the
-input file discussed in {ref}`5.2.1][], simulating convection in a
+input file discussed in {ref}`5.2.1`[], simulating convection in a
```
Comment (MFraters) [inline doc/sphinx/user/run-aspect/visualizing-results/stat-data.md:9]:
```suggestion
input file discussed in {ref}`5.2.1`, simulating convection in a
```

**Review by MFraters — COMMENTED** [review]
You missed one, otherwise looks good. The indenter complains, so please indent. I am wondering whether it makes sense for the indenter to indent markdown. Any opinions on that?

**Review by MFraters — APPROVED** [review]
See #4694 for discussion on the brackets.

File: doc/sphinx/user/run-aspect/visualizing-results/stat-data.md [hunk 2]
```diff
@@ -6,11 +6,10 @@ In addition to the graphical output discussed above,
...
 box. After running ASPECT, you will find a file
 called `statistics` in the output directory that, at the time of writing this,
-looked like this: This file has a structure that looks (at the time of writing
-this section) like this:
+looked like this:
```
Comment (gassmoeller) [inline doc/sphinx/user/run-aspect/visualizing-results/stat-data.md:12]:
```suggestion
looked like this:
```

File: doc/sphinx/user/run-aspect/visualizing-results/stat-data.md [hunk 3]
```diff
@@ -176,16 +175,18 @@ manual at <http://www.gnuplot.info/>. You can also get a good amount of
...
 at a later time. Speaking from experience, you will not believe how often one wants to tweak a figure long after it was first
 created. In such situations it is outstandingly helpful if one still has both the actual data as well as the script that generated
 the graphic.
+
+:::{admonition} TODO
+Footnote: reference to A.127; parameter sections not set up so this reference does not make sense.
+:::
+
+
+
```
Comment (gassmoeller) [inline doc/sphinx/user/run-aspect/visualizing-results/stat-data.md:192]:
```suggestion

```

**Pull request merged**